api/src/services/log-docs.js

```diff
@@ -0,0 +1,70 @@
+const moment = require('moment');
```

Comment:
I don't think this needs to be this generic right now. We can easily refactor it to be generic later if it makes sense. I think it'd make more sense for this service to be called `replication-limit` (or similar). This will mean a lot of the functions can be renamed to be much simpler, eg: `get` and `put`.

Reply:
OK, I'm going to name it `replication-limit-log`

Reply:
Fixed.